.gitignore

```diff
@@ -22,7 +22,7 @@ summa.exe
 .DS_Store
 *_local
 *log
+summa.exe.dSYM*
 # makefile
-*-local
```

Comment:
Just leave this in as well - some of us may be using `*-local` rather than `*_local`